shared/test/integration/tbpl-manifest.json

```diff
@@ -33,8 +33,6 @@
     "apps/settings/test/marionette/tests/app_permission_settings_test.js": "Bug 1004348 - Disable intermittent failing test, manipulate app permissions set geolocation of first app to Grant",
     "apps/settings/test/marionette/tests/hotspot_wifi_settings_test.js": "Bug 1006527 - Disable intermittent failing travis test, manipulate hotspot wifi settings resets settings for second load",
     "apps/settings/test/marionette/tests/message_settings_test.js": "",
-    "apps/sms/test/marionette/composer_test.js": "Bug 1121766 - TEST-UNEXPECTED-FAIL | Messages Composer Messages Composer Test Suite Message char counter and MMS label",
```

Comment:
Sorry for the confusion, it definitely won't go to the final patch, I just wanted to see if it helps for composer_test intermittent failures as well - yes it helps, but not for everything. So composer_test will be enabled in bug 1121766 with additional change (see https://github.com/mozilla-b2g/gaia/pull/27474 and https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=db4af079b192)